atlp-rwanda/trojans-ec-fe
# #184760067 - Track order status

What does this PR do?
A buyer should be able to track their order status

Description of Task to be completed?
 A Buyer should be able to track the status of an order of their purchase and plan accordingly.

Acceptance Criteria:

 GIVEN that the buyer is on the order history page
    - WHEN the buyer clicks on an order
    - THEN the buyer should be taken to an order detail page displaying the order status
    - GIVEN that the buyer is on the order detail page
    - WHEN the buyer refreshes the page
    - THEN the order status should be updated in real-time
    - GIVEN that the buyer is on the order detail page
    - WHEN the buyer clicks on a button to contact customer support
    - THEN the buyer should be directed to the customer support page.

How should this be manually tested?
Login as a buyer ;purchase a product, check our dashboard and click on orders to view the order status.

Any background context you want to provide?
N/a

What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2628461

Screenshots
![Uploading Screenshot 2023-05-11 at 21.28.13.png…]()
![Uploading Screenshot 2023-05-11 at 21.28.33.png…]()
